### PR TITLE
Fix Travis CI configuration

### PR DESCRIPTION
**Pull request by swissspidy**
* Exit immediately if one of the commands returns a non-zero status  
  It looks like Travis isn't failing despite errors, see https://travis-ci.com/github/google/web-stories-wp/jobs/300261039 for an example.
* Fix Composer configuration
* Fix E2E tests configuration
* Fix two accessibility issues in the editor that were flagged by the E2E tests

### CODE REVIEW

**Comment by swissspidy** [comment]
It worked! See https://travis-ci.com/github/google/web-stories-wp/jobs/300265885

File: assets/src/edit-story/components/canvas/carousel/index.js [hunk 1]
```diff
@@ -121,7 +121,9 @@ const List = styled(Area).attrs({
   }
 `;
 
-const Li = styled.li`
+const Li = styled.li.attrs({
```
Comment (spacedmonkey) [inline assets/src/edit-story/components/canvas/carousel/index.js:124]: Why was this added?

File: .travis.yml [hunk 2]
```diff
@@ -39,6 +39,7 @@ env:
     - PUPPETEER_SKIP_CHROMIUM_DOWNLOAD=true
 
 install:
+  - set -e
```
Comment (spacedmonkey) [inline .travis.yml:42]: What does this do?

File: assets/src/edit-story/components/canvas/carousel/index.js (hunk 1 above)
Comment (swissspidy) [reply assets/src/edit-story/components/canvas/carousel/index.js:124]: As the commit says: listbox items need to have role `option`. The carousel is a listbox.

File: .travis.yml (hunk 2 above)
Comment (swissspidy) [reply .travis.yml:42]: As per the [Bash manual](https://www.gnu.org/software/bash/manual/html_node/The-Set-Builtin.html):

> Exit immediately if a pipeline (see Pipelines), which may consist of a single simple command (see Simple Commands), a list (see Lists), or a compound command (see Compound Commands) returns a non-zero status.

Before this, the individual scripts within these sections could fail without causing Travis to fail the build.
